data/cargo_contributions_bench_sample.csv

```diff
@@ -0,0 +1,10000 @@
+ID,MAINTAINER,REPO,CONTRIBUTIONS,NAME
+30742,github@aepsil0n,https://github.com/aepsil0n/acacia,118,acacia
```

Comment:
Thinking out loud -- why we don't simply read the first 10k lines out of the _whole_ file during the benches, rather than create a separate file for it?

Concretely, what I am proposing is something like this (in pseudo-Rust):

```rust
let mut file = File::open("data/cargo_contributions.csv").unwrap();
file.take(10000); // take only the first 10k lines
...
```